trade-finance-manager-api/src/v1/controllers/acbs.controller.js

```diff
@@ -138,13 +156,36 @@ const issueAcbsFacilities = async (deal) => {
       ...deal.exporter,
     },
   }));
-
   const acbsIssuedFacilities = await Promise.all(acbsIssuedFacilitiesPromises);
 
-  const acbsIssueFacilityTasks = await Promise.all(
+  return Promise.all(
     acbsIssuedFacilities.map((acbsTaskLinks) => addToACBSLog({ acbsTaskLinks })),
   );
-  return acbsIssueFacilityTasks;
+};
+/**
+ * Amend facility controller function responsible for invoking
+ * respective API and writes ACBS task links to DB.
+ * @param {Object} amendments Facility amendments object
+ * @param {Object} facility Complete TFM facility object
+ * @param {Object} deal Bespoke deal object
+ */
+const amendAcbsFacility = async (amendments, facility, deal) => {
+  let payload;
+
+  // TO-DO : EPOCH Convergence
+  if (amendments.coverEndDate) {
+    payload = formatCoverEndDate(amendments);
+  }
+
+  api.amendACBSfacility(payload, facility, deal)
```

Comment:
Isn't this one supposed to have `await`?

Reply:
No as I am resolving the promise with `then`.